best_practices/business-logic.rst

```diff
@@ -57,8 +57,9 @@ post URL. Let's create a new ``Slugger`` class inside ``src/Utils/``::
     }
 
 If you're using the :ref:`default services.yaml configuration <service-container-services-load-example>`,
-this class is auto-registered as a service whose ID is ``App\Utils\Slugger`` (or
-simply ``Slugger::class`` if the class is already imported in your code).
+this class is auto-registered as a service whose ID is ``App\Utils\Slugger`` (as
+an alternative you can use ``Slugger::class``, if the class is already imported
```

Comment:
i would change the paranthesis to "(to prevent against typos, import the class and write `Slugger::class` in your code)"